experiments/exp1331_swe_smith_marin.py

```diff
@@ -0,0 +1,83 @@
+"""
+#1331: Marin SWE-Agent
+
+GitHub Issue: https://github.com/marin-community/marin/issues/1331
+"""
+
+import dataclasses
+
+from levanter.data.text import ChatLmDatasetFormat
+from levanter.layers.rotary import YarnRotaryEmbeddingsConfig
+
+from experiments.defaults import default_sft, default_tokenize
+from experiments.llama import llama_8b
+from experiments.tootsie.exp916_tootsie_spoonbill_cooldown import spoonbill_zloss_tulu3_sft_config
+from marin.download.huggingface.download import DownloadConfig, download
+from marin.execution.executor import ExecutorStep, executor_main, this_output_path, versioned
+from marin.resources import TpuPodConfig
+
+sft_experiments = []
+swe_smith_sft_config = dataclasses.replace(
+    spoonbill_zloss_tulu3_sft_config,
+    learning_rate=1e-4,
+    num_train_steps=600,
+    max_seq_len=32768,
+    train_batch_size=16,
```

Comment:
LGTM! Mainly was just checking the SFT experiment settings against the config I used w/ `torchtune` (located [here](https://github.com/SWE-bench/SWE-smith/blob/main/configs/train/full_ft_qwen_7b.yml)).